lib/exceptions.h

```diff
@@ -46,18 +46,26 @@ class MYSQLPP_EXPORT Exception : public std::exception
 {
 public:
 	/// \brief Create exception object as copy of another
+#if __cplusplus >= 201103L
+    Exception(const Exception&) MAY_THROW() = default;
+#else
 	Exception(const Exception& e) MAY_THROW() :
 	std::exception(e),
 	what_(e.what_)
 	{
 	}
+#endif
 
 	/// \brief Assign another exception object's contents to this one
-	Exception& operator=(const Exception& rhs) throw()
+#if __cplusplus >= 201103L
+	Exception& operator=(const Exception& rhs) noexcept = default;
+#else
+    Exception& operator=(const Exception& rhs) throw()
 	{
 		what_ = rhs.what_;
 		return *this;
 	}
+#endif
```

Comment:
I'll only accept this sort of thing if you can show me Compiler Explorer output showing that the compiler generates something materially different when you do this and that, under optimization, the result is faster. Otherwise, it's just making the code shorter by making it longer.